kroma-validator/cmd/main.go

```diff
@@ -27,7 +25,7 @@ func main() {
 	app := cli.NewApp()
 	app.Flags = cliapp.ProtectFlags(flags.Flags)
 	app.Version = fmt.Sprintf("%s-%s", Version, Meta)
-	app.Name = "validator"
+	app.Name = "kroma-validator"
```

Comment:
I reverted it since the command name is determined by docker-compose build target.